inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp

```diff
@@ -41,84 +45,126 @@ void OPCache::update_ops_cache(const std::shared_ptr<ngraph::Function> &func, co
         }
         update_ops_cache(op, source_model);
     }
+    std::cout << "\t" << m_ops_cache.size() - cached_ops_count << " new OPs were cached." << std::endl;
 }
 
 void OPCache::serialize_cached_ops(const std::string &serialization_dir) {
     if (!CommonTestUtils::directoryExists(serialization_dir)) {
         CommonTestUtils::createDirectoryRecursive(serialization_dir);
     }
     for (const auto &op : m_ops_cache) {
-        try {
-            auto rt_info = op.first->get_rt_info();
-            std::cout << "Serializing function wrapping op " << op.first << std::endl;
-            if (rt_info.find("source_model") != rt_info.end()) {
-                auto val = rt_info["source_model"];
-                auto source_model = std::dynamic_pointer_cast<ngraph::VariantWrapper<std::string>>(val);
-                if (source_model != nullptr) {
-                    std::cout << " Taken from model: " << source_model->get() << std::endl;
-                }
+        auto res = serialize_function(op, serialization_dir);
+        if (res != FAILED) {
```

Comment:
I thought about it, it could be some utility function which consumes function pointer which should be run with retries, but signature if received function could be different case by case, and also as you can see here we have some more advanced logic when retries happen (only if some substring were found in exception message)